cmd/docker-app/render.go

```diff
@@ -68,5 +69,6 @@ func renderCmd(dockerCli command.Cli) *cobra.Command {
 	cmd.Flags().StringArrayVarP(&renderSettingsFile, "settings-files", "f", []string{}, "Override settings files")
 	cmd.Flags().StringArrayVarP(&renderEnv, "set", "s", []string{}, "Override settings values")
 	cmd.Flags().StringVarP(&renderOutput, "output", "o", "-", "Output file")
+	cmd.Flags().StringVar(&formatDriver, "formatter", "yaml", "Configure the output format")
```

Comment:
Are we considering adding more format options, or only choose between `yaml` and `json`? Otherwise we may want to use the same syntax as we do for commands on the docker cli;

```
--format '{{json .}}'

--format '{{json .Some.Key}}'
```

Reply:
I think you're thinking about a different thing - `--format` here is not designed to extract specific parts of the data structure, but only to indicate what format the output will be presented in. As far as other options are concerned, this was part of the acceptance criteria on the internal tracker:

> ### Acceptance criteria
>
> * Choose to render output as JSON
> * UI flexibility to output other formats in the future